client/src/pages/global.css

```diff
@@ -35,6 +35,37 @@ p {
     padding: 0;
 }
 
+form:not(#logout) {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    margin: var(--spacing-large);
+    padding: var(--spacing-normal);
+    row-gap: var(--spacing-small);
+}

```

Comment:
This exception seems like it should be overridden at the component-level rather than the global-level.